### PR TITLE
feat: Can we provide a better review assignment email subject. #3760

### PR DESCRIPTION
**Pull request by evyncke**
About #3760 

1/ email subject now contains the reviewer name + deadline

2/ copies is also sent to the review requester and also to all secretaries

As discussed in #3760 with @cjbc @Abhayakara @kivinen @jgscudder

### CODE REVIEW

**Comment by evyncke** [comment]
It looks like that I should also have updated the tests...

**Comment by rjsparks** [comment]
> It looks like that I should also have updated the tests...

Do you want me to do that, or will you have time soon?

**Comment by evyncke** [comment]
Happy to keep learning on the tests aspects in the coming weeks.

File: ietf/review/utils.py [hunk 1]
```diff
@@ -439,9 +439,9 @@ def assign_review_request_to_reviewer(request, review_req, reviewer, add_skip=Fa
 
     email_review_request_change(
         request, review_req,
-        "%s %s assignment: %s" % (review_req.team.acronym.capitalize(), review_req.type.name,review_req.doc.name),
+        "For %s, %s %s review by %s assigned: %s" % (reviewer.person.ascii, review_req.team.acronym.capitalize(), review_req.type.name, review_req.deadline, review_req.doc.name),
```
Comment (larseggert) [inline ietf/review/utils.py:442]: Why ASCII?

Comment (evyncke) [reply ietf/review/utils.py:442]: Good point, could be reviewer.person.name and have my nice "Éric" ;-)

**Comment by rjsparks** [comment]
Converting to draft while the above changes are in progress. Please mark it ready for review once they're in place.

**Comment by evyncke** [comment]
To clean up the PR (messed up by wrong merge), I have created a new one #5415

**Pull request closed without merging**